### DISCUSSION TITLE
Creating a Custom ArchRGenome: what about transcripts?

### DISCUSSION THREAD

**Discussion by tao-bioinfo** [GreenleafLab/ArchR#893] [Questions / Documentation] [answered]
In https://www.archrproject.com/bookdown/getting-set-up.html, it says creating a geneAnnotation object from:

A “genes” object
An “exons” object
A GRanges object containing standed transcription start site (TSS) coordinates.

What about transcripts? Generally, a gene contains multiple transcripts, and a transcript contain multiple exons.

**Comment by rcorces** [comment, marked as answer]
By default, ArchR deals with genes for simplicity. If you really care about transcripts, presumably you can create a custom genome where the transcripts each have unique names and are treated as if they are individual genes. I've never done this but it should work. The important part is that the entry in the `geneAnnotation` object is called "genes"

**Comment by rcorces** [comment]
Also please note that Issues are only for bugs. Discussions is where questions and feature requests belong.
I'm transferring your question there but please follow this in the future.

![image](https://user-images.githubusercontent.com/10343046/125661272-37bf1598-b843-4414-ac89-eed118cfa9d1.png)

**Reply by tao-bioinfo** [reply]
Thanks. I'll have a try then
